maps/DeadSpace/M.02 Ishimura (M).dmm

```diff
@@ -654,7 +654,7 @@
 "amD" = (/obj/machinery/door/firedoor/border_only,/obj/structure/disposalpipe/segment,/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/structure/cable/green{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/door/airlock/security{name = "Engineering Security Station"},/obj/machinery/door/blast/shutters{dir = 1; icon_state = "shutter0"; id = "e_security"},/turf/simulated/floor/tiled/monotile,/area/ishimura/upper/security/checkpoint/engineering)
 "amE" = (/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 8},/turf/simulated/floor/tiled/steel_grid,/area/ishimura/upper/engineering/hub)
 "amF" = (/turf/simulated/floor/tiled,/area/ishimura/upper/engineering/lobby)
-"amG" = (/obj/effect/wingrille_spawn/reinforced,/obj/machinery/door/blast/shutters{dir = 1; icon_state = "shutter0"; id = "e_security"},/obj/machinery/door/firedoor/border_only,/obj/structure/cable/green{d2 = 4; icon_state = "0-4"},/turf/simulated/floor/plating,/area/ishimura/upper/security/checkpoint/engineering)
```

Comment:
what are the mapping changes?

Reply:
Windows were largely overhauled. These window types being the singular windows, not the full window sprites. I've unified them as one-use on the map.